packages/admin/cms-admin/src/dashboard/DashboardScopeIndicator.tsx

```diff
@@ -0,0 +1,31 @@
+import { messages } from "@comet/admin";
+import { Domain } from "@comet/admin-icons";
+import { Typography } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+import { ContentScopeIndicator } from "../contentScope/ContentScopeIndicator";
+
+export const DashboardScopeIndicator = () => {
+    return (
+        <ContentScopeIndicator global>
```

Comment:
I believe the dashboard will be scope-aware in the near future, especially features such as latest content updates. We should leave the scope indicator in the app.